lib/provider/camunda/parts/implementation/ErrorEventDefinition.js

```diff
@@ -13,52 +15,73 @@ module.exports = function(
   var getValue = function(modelProperty) {
```

Comment:
In ErrorEventDefiniton.js we use getValue and setValue helpers, we don't do that in EscalationEventDefinitions.js.

Is there a reason for that (it is in principle doing the same thing, isn't it)? If not, we should do it the same way in both files.

Reply:
The only thing is that the escalation one is single use, this one is called twice.
The other examples of validation aware text fields don't use generic getters and setters, so this one is sort of the odd one.
I can align it regardless if you think it's better.

Reply:
well when ErrorEventDefinitions is the only one, where we call them twice, that is a IMO a valid reason why we have separate helper functions for getting and setting :+1: